tests/unit/components/filter-selector/FilterSelector.spec.jsx

```diff
@@ -0,0 +1,46 @@
+import { render, screen } from '@testing-library/react'
+import user from '@testing-library/user-event'
+import FilterSelector from '~/components/filter-selector/FilterSelector'
+
+vi.mock('simplebar-react', () => {
+  return {
+    __esModule: true,
+    default: function MockedSimpleBar(props) {
+      return <div>{props.children}</div>
+    }
+  }
+})
+
+vi.mock('react-i18next', () => ({
```

Comment:
I guess you don`t need this mock, you can find element using screen.getByText('common.noItems') and then check if this element is in document with .toBeInTheDocument()

Reply:
without this mock the test passed but with an error Error: Not implemented: window.computedStyle(elt, pseudoElt). 
This mock allows me to avoid the error. What do you suggest in this case?